### DISCUSSION TITLE
pass in additional request params to the endpoint?

### DISCUSSION THREAD

**Discussion by kay-hal** [FlowiseAI/Flowise#235] [Q&A] [answered]
is there a way to pass in additional requests params to the endpoint and use that metadata as a parameter in, say, url loader?
Another reason I'd need this is to namespace the vector store based on some user_uuid from my app

**Comment by HenryHengZJ** [comment]
yeah, you have the option to pass the metadata to the body:
<img width="655" alt="image" src="https://github.com/FlowiseAI/Flowise/assets/26460777/0b79d7a3-1215-4f97-b1f1-b06f63e04943">

So smtg like this:
<img width="488" alt="image" src="https://github.com/FlowiseAI/Flowise/assets/26460777/e468ca86-488d-4f0f-9230-0aa6126f3d18">

**Reply by kay-hal** [reply]
Ok awesome. And how would I reference it in the flow?

**Reply by HenryHengZJ** [reply, marked as answer]
You can refer to one of the templates in marketplace - Metadata Upsert or Metadata Load
